modules/javafx.web/src/test/java/test/javafx/scene/web/MiscellaneousTest.java

```diff
@@ -183,6 +183,8 @@ public void call(long createdTime, long interval, int index) {
             // Try various intervals
             for (int i = 0; i < timer.INTERVAL_COUNT; i++) {
                 int timeout = i * (1000 / timer.INTERVAL_COUNT);
+                // Webkit recomends minimum timeout value should be 10
+                if(timeout == 0) timeout = 10;
```

Comment:
Are you sure that the intermittent failures happen when the timeout value is 0? Based on the assertion message in the bug report, I didn't think this was the case, but I might be misinterpreting it.